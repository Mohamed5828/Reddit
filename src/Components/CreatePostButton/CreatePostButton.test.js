import {shallow} from 'enzyme';
import { BrowserRouter, Routes } from "react-router-dom/dist";

// import Components
import CreatePostButton from "./CreatePostButton";

describe("Create Post button Component", () => {
    it("this is a test for Create post button", () => {
        expect(
          shallow(
            <BrowserRouter>
              <Routes>
                <CreatePostButton />
              </Routes>
            </BrowserRouter>
          )
        ).toMatchSnapshot();

    });
});