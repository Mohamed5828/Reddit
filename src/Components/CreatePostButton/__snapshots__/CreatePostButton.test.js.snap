// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Post button Component this is a test for Create post button 1`] = `
<Router
  location={
    Object {
      "hash": "",
      "key": "default",
      "pathname": "/",
      "search": "",
      "state": null,
    }
  }
  navigationType="POP"
  navigator={
    Object {
      "action": "POP",
      "createHref": [Function],
      "go": [Function],
      "listen": [Function],
      "location": Object {
        "hash": "",
        "key": "default",
        "pathname": "/",
        "search": "",
        "state": null,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Routes>
    <CreatePostButton />
  </Routes>
</Router>
`;
